.gitignore

```diff
@@ -0,0 +1,5 @@
+newindex.html;
+oldapp.js;
+index.html;
+app.js;
+sales.html
```

Comment:
app.js and sales.html probably shouldn't be in the gitignore file since you want those to be tracked by git.